app/views/merchant/items/index.html.erb

```diff
@@ -5,6 +5,9 @@
     <h3><%= item.name %></h3>
     <p>Price: <%= number_to_currency(item.price) %></p>
     <p>Inventory: <%= item.inventory %></p>
+    <% if item.no_orders? %>
```

Comment:
Love that you used a method here; much better than item.orders.empty? living in the view